interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -510,7 +510,7 @@ MathFuncts, errGen) {
   function SampleBinCtrl($scope, SampleBin, MlModelTracker) {
     // give our templates a way to access the SampleBin service
     $scope.sb = SampleBin;
-    $scope.modelInfo = MlModelTracker;
+    SampleBin.mlModelInfo = MlModelTracker;
```

Comment:
I think I see what you're saying, and I agree. The `MlModelTracker` line got added during @dongbohu's big retro-fit and he had attached `MlModelTracker` directly to the `$scope` in the same way he did for other pages.

To get that retro-fit to work properly with the original design for this module, `SampleBin` really needs to have a handle to `MlModelTracker`, which is why I changed that. If we think of setting `SampleBin.mlModelInfo` as an "initialization" step, then swapping the lines the way you suggest does make better sense.

I'll swap it around. This might change with `sampleBin` refactoring, but it's good to be thinking about it this way.